components/custom_status/custom_status_modal.tsx

```diff
@@ -46,6 +50,7 @@ const CustomStatusModal: React.FC<Props> = (props: Props) => {
     const [showEmojiPicker, setShowEmojiPicker] = useState<boolean>(false);
     const [text, setText] = useState<string>(currentCustomStatus.text);
     const [emoji, setEmoji] = useState<string>(currentCustomStatus.emoji);
+    const [expiry, setExpiry] = useState<string>('4 hours');
```

Comment:
Should we store `expiryMenuItems[].id` here?

E.g. `four-hours` instead of `4 hours`

Reply:
The value used here is shown on the screen. If we use id here then we have to iterate the `expiryMenuItems` arr to find the corresponding text. And what's the benefit of storing id?

Reply:
What if we use 

```js
const expiryMenuItems = {
        'dont-clear': {
            text: "Don't clear",
            localizationId: 'dont_clear',
            id: 'dont-clear',
        },
        'thirty-minutes': {
            text: '30 minutes',
            localizationId: 'thirty_minutes',
            id: 'thirty-minutes',
        },
        'one-hour': {
            text: '1 hour',
            localizationId: 'one_hour',
            id: 'one-hour',
        },
        'four-hours': {
            text: '4 hours',
            localizationId: 'four_hours',
            id: 'four-hours',
        },
        today: {
            text: 'Today',
            localizationId: 'today',
            id: 'today',
        },
        'this-week': {
            text: 'This week',
            localizationId: 'this_week',
            id: 'this-week',
        },
    };
```

Reply:
Yeah, we can do this, but why do we want to store id in the state?